app/utils/SQLHelpers.scala

```diff
@@ -39,7 +41,14 @@ object ObjectId extends FoxImplicits {
   implicit val jsonFormat = Json.format[ObjectId]
   def fromBsonId(bson: BSONObjectID) = ObjectId(bson.stringify)
   def generate = fromBsonId(BSONObjectID.generate)
-  def parse(input: String) = BSONObjectID.parse(input).map(fromBsonId).toOption.toFox ?~> Messages("bsonid.invalid", input)
+  def parse(input: String) = parseSync(input).toFox ?~> Messages("bsonid.invalid", input)
+  private def parseSync(input: String) = BSONObjectID.parse(input).map(fromBsonId).toOption
+
+  def stringObjectIdReads(key: String) =
```

Comment:
this is still used?

Reply:
That’s right, I was originally going to do more in this PR. I intend to use it instead of the one below in all places eventually.

Reply:
Let's do this in the respective PR then.